src/main/java/org/broadinstitute/hellbender/utils/dataflow/BucketUtils.java

```diff
@@ -192,4 +191,35 @@ public static boolean fileExists(String path, PipelineOptions popts) {
         }
         return true;
     }
+
+    /**
+     * Create a new GCS options with a given api key.
+     * @param apiKey a valid GCS api key
+     */
+    public static GCSOptions getAuthenticatedGCSOptions(String apiKey) {
```

Comment:
Yes, this method was removed in the rebase of the original BQSR spark branch at the request of @jean-philippe-martin in favor of `SparkCommandLineProgram.getAuthenticatedGCSOptions()` -- should be deleted here.